app/managers/websocket_manager.ts

```diff
@@ -92,23 +99,21 @@ class WebsocketManager {
     public closeAll = () => {
         for (const url of Object.keys(this.clients)) {
             const client = this.clients[url];
-            if (client.isConnected()) {
-                client.close(true);
-                this.getConnectedSubject(url).next('not_connected');
-            }
+            client.close(true);
```

Comment:
There is no problem on closing the same client twice, so better safe than sorry (and also makes sure the reconnect timeout doesn't run).